docker-compose.visualize.yml

```diff
@@ -32,10 +31,22 @@ services:
     command: |
       sh -c "/bin/tar -xf /kbn_network.tar.gz -C /usr/share/kibana/plugins;
              /usr/local/bin/kibana-docker --allow-root;"
-
-    logging: *default-logging
+    logging:
+      driver: syslog
+      options:
+        syslog-address: "udp://localhost:8514"
   visualize-scripts:
     image: semtech/mu-scripts:1.0.0
     volumes:
       - ./scripts/visualize-scripts:/app/scripts/
     restart: "no"
+  log-alert:
```

Comment:
not sure what we think about directly adding this to the visualize docker compose file... do you think it should be in a separate file?